server/internal/milvus/milvus.go

```diff
@@ -100,3 +100,12 @@ func (s *S) UpdateVectorStoreName(ctx context.Context, oldName, newName string)
 func (s *S) DeleteVectorStore(ctx context.Context, name string) error {
 	return s.client.DropCollection(ctx, name)
 }
+
+// InsertDocuments inserts documents into a collection in milvus.
+func (s *S) InsertDocuments(ctx context.Context, name string, vectors [][]float32) error {
+	vectorCol := entity.NewColumnFloatVector(vectorColName, len(vectors[0]), vectors)
+	if _, err := s.client.Insert(ctx, name, "", vectorCol); err != nil {
```

Comment:
It's nice to have some comment the empty string that is passed to `Insert` as other people don't need to read the client code.